doc/sphinx/practical-tools/coqide.rst

```diff
@@ -330,3 +330,192 @@ If you choose something else than UTF-8, then missing characters will
 be written encoded by `\x{....}` or `\x{........}` where each dot is
 an hexadecimal digit: the number between braces is the hexadecimal
 Unicode index for the missing character.
+
+.. _coqide-debugger:
+
+Debugger
+--------
+
+Version 8.15 introduces a visual debugger for |Ltac| tactics within
+CoqIDE.  It supports setting breakpoints visually and automatically
+displaying the stopping point in the source code with "continue",
+"step over" "step in" and "step out" operations.  The call stack and variable
+values for each stack frame are shown in a new panel.
+
+The debugger is based on the non-visual |Ltac| :ref:`debugger <interactive-debugger>`.
+We'd like to eventually support other scripting facilities such as Ltac2.
+
+Since the visual debugger is new in 8.15, you may encounter bugs or usability issues.
+The behavior and user interface will evolve as the debugger is refined.
+There are notes on bugs and potential enhancements at the end of
+`this page <https://github.com/coq/coq/wiki/Ltac-Debugger-Preview>_`.
+Feel free to suggest changes and improvements by opening an issue on
+`GitHub <https://github.com/coq/coq/issues/new>_`, or contact `@jfehrle`
```

Comment:
Sorry the syntax is so weird that I mixed things up and did not suggest the correct fix there (even though I had in another place) :laughing: 
The backtick must be before the `_`.
```suggestion
`this page <https://github.com/coq/coq/wiki/Ltac-Debugger-Preview>`_.
Feel free to suggest changes and improvements by opening an issue on
`GitHub <https://github.com/coq/coq/issues/new>`_, or contact `@jfehrle`
```